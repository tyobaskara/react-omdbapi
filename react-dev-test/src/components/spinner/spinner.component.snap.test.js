import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from './spinner.component';

test('Spinner snaptest', () => {
  const component = renderer.create(
    <Spinner/>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
