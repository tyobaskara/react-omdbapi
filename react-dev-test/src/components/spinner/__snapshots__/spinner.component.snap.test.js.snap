// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner snaptest 1`] = `
<div
  className="spinner-overlay"
>
  <div
    className="spinner-container"
  />
</div>
`;
